app/components/datasources/DataConnections.js

```diff
@@ -668,12 +681,12 @@ DataConnections.propTypes = {
   patient: PropTypes.shape({
     dataSources: PropTypes.oneOf([PropTypes.shape(clinicPatientDataSourceShape), PropTypes.shape(userDataSourceShape)])
   }),
-  shownProviders: PropTypes.arrayOf(PropTypes.oneOf(activeProviders)),
+  shownProviders: PropTypes.arrayOf(PropTypes.oneOf(availableProviders)),
   trackMetric: PropTypes.func.isRequired,
 };
 
 DataConnections.defaultProps = {
-  shownProviders: activeProviders,
+  shownProviders: availableProviders,
```

Comment:
By default, all available providers _can_ be shown.  This does not mean they will be, they still have to also be present in the result of `getActiveProviders`